src/server.ts

```diff
@@ -1,13 +1,31 @@
 import dotenv from 'dotenv';
 dotenv.config();
-
 import express from 'express';
+import { addGear, readGear, deleteGear } from './utils/gear';
+import type { Gear } from './types';
+import { connectDatabase } from './utils/database';
 
 const app = express();
 const port = process.env.PORT || 3001;
 
 app.use(express.json());
 
+app.post('/api/gear', async (request, response) => {
+  const gear: Gear = request.body;
+  await addGear(gear);
+  return response.status(200).send(gear);
+});
+
+app.get('/api/gear', async (_request, response) => {
+  const gear = await readGear();
+  response.json(gear);
+});
+
+// app.delete('/api/gear/:name', async (request, response) => {
+//   const name = request.params;
+//   await deleteGear, response.status(200).send('Deleted');
```

Comment:
These are two lines in one, you can't separate code that way with a comma

Reply:
Unfortunately, fixing this doesn't do anything to fix the global issue.